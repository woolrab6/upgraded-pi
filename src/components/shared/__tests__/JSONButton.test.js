import React from 'react'
import {configure, shallow} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import JSONButton from '../JSONButton'

configure({adapter: new Adapter()})

it('renders button with given url', () => {
  const url = 'https://somebackend.xyz/resource/12345'

  const btn = shallow(<JSONButton url={url} />)
  expect(btn.props().url).toEqual(url)

  expect(btn.getElements()).toMatchSnapshot()
})
