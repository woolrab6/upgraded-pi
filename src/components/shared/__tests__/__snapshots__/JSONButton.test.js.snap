// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders button with given url 1`] = `
Array [
  <BackendResourceBadgeButtonWithResourceModal
    label="JSON"
    url="https://somebackend.xyz/resource/12345"
  />,
]
`;
